pkg/config/config.go

```diff
@@ -828,6 +828,9 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("runtime_security_config.remote_tagger", true)
 	bindEnvAndSetLogsConfigKeys(config, "runtime_security_config.endpoints.")
 
+	// Serverless Agent
+	config.BindEnvAndSetDefault("send_logs", true)
```

Comment:
Maybe we can use a more explicit name to avoid confusion. Eg "lambda_logs_enabled"